### ISSUE TITLE
Unit Test Error on `merge:`

### ISSUE THREAD

**Issue by bobspryn** [ReactiveCocoa/ReactiveCocoa#1323]
Trying to hunt this one down. If anyone can spot something I would appreciate it.

I have a simple unit test on a view model. In the very beginning of `init` in that view model is this:

```
RAC(self, currentLocation) = [[[self.locationManager rac_locationUpdateSignal]
    map:^id(NSArray *locations) {
        return [locations lastObject];
    }]
    catchTo:[RACSignal return:nil]];
```

`rac_locationUpdateSignal` is a category on `CLLocationManager`

```
- (RACSignal *) rac_locationUpdateSignal {
    self.delegate = self;
    RACSignal *signal = objc_getAssociatedObject(self, _cmd);
    if (signal != nil) return signal;
    RACSignal *didUpdate = [[self rac_signalForSelector:@selector(locationManager:didUpdateLocations:) fromProtocol:@protocol(CLLocationManagerDelegate)]
        map:^id(RACTuple *values) {
            return values.second;
        }];
    RACSignal *didFail = [[self rac_signalForSelector:@selector(locationManager:didFailWithError:) fromProtocol:@protocol(CLLocationManagerDelegate)]
        mapReplace:nil];
    signal = [RACSignal merge:@[didUpdate, didFail]];
    objc_setAssociatedObject(self, _cmd, signal, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
    return signal;
}
```

In unit tests only, I end up with this error:
`*** Assertion failure in __36-[RACStream(Operations) flattenMap:]_block_invoke_2(), /Users/bobspryn/lohi/Three-Cents-iOS/Pods/ReactiveCocoa/ReactiveCocoaFramework/ReactiveCocoa/RACStream.m:75`

Basically as soon as the first location manager signal is sent in `merge` it throws that error. Can't figure out why.

**Comment by amccarri** [comment]
I'm seeing this and I do not have it linked twice unless CocoaPods is doing something weird?  I am building a framework though, not an app.

**Comment by saiday** [comment]
@sprynmr Hi, sprynmr. Can you explain more on `basing the configuration file`?
Thanks
